src/OmnichannelChatSDK.ts

```diff
@@ -458,6 +458,20 @@ class OmnichannelChatSDK {
         return this.chatToken;
     }
 
+    public async getCallingToken(): Promise<string> {
+        if (this.chatToken && Object.keys(this.chatToken).length === 0) {
+            return '';
+        }
+
+        if (this.chatToken.voiceVideoCallToken) {
+            console.log(`calling:acs`);
```

Comment:
Do we have a telemetry strategy defined for the sdk? Logging to the console is only visible to end users.